### DISCUSSION TITLE
Migration of existing project with Cutelyst3

### DISCUSSION THREAD

**Discussion by cezaryece** [cutelyst/cutelyst#449] [Q&A]
Any warnings, boilerplates or quicksand while moving existing project to Cutelyst 4?
Is there something more to do than change in CMakeList.txt line
find_package(Cutelyst3Qt6 REQUIRED)
to
find_package(Cutelyst4Qt6 REQUIRED)

And more urgent question: can I have installed Cutelyst3 and Cutelyst4 side by side?

**Comment by dantti** [comment]
yes, you can have both installed side by side, everything should be versioned.

Besides this cmake change you might need to make changes to your usage of the Header class it's the one that changed the most.

You can look at Virtlyst, pastlyst and CMlyst what was changed when migrating to v4.
